openquakeplatform_building_class/models.py

```diff
@@ -60,8 +63,27 @@ class OCCUP_TYPE(object):
     (OCCUP_TYPE._GOVERNMENTAL, 'governmental')
     )
 
+
+BOOL_CHOICES = ((True, 'Yes'), (False, 'No'))
 class UserSettings(SingleOwnerMixin, models.Model):
-    publish_info = models.BooleanField()
+    name = models.CharField(_('Individual Name'), max_length=255, blank=True, null=True,
+                            help_text=_('name of the responsible person surname, given name,'
+                                        ' title separated by a delimiter'))
+    organization = models.CharField(_('Organization Name'), max_length=255,
+                                    blank=True, null=True,
+                                    help_text=_('name of the responsible organization'))
+    position = models.CharField(_('Position Name'), max_length=255, blank=True, null=True,
+                                help_text=_('role or position of the responsible person'))
+    publish_info = models.BooleanField(choices=BOOL_CHOICES, help_text=_('''Would you like for your contribution to be acknowledged in future documentation (eg. technical reports, scientific papers in peer reviewed journals, ...) ?<br>
```

Comment:
"Would you like for your contribution" should be "Would you like your contribution" (i.e. no "for")